challenges/objects-arrays.js

```diff
@@ -13,6 +13,41 @@
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceious
 
 // Using your dinosaur objects, log answers to these questions:
+function Dinosaur (dinosaur_attributes) {
+  this.name = dinosaur_attributes.name;
+  this.diet = dinosaur_attributes.diet;
+  this.weight= dinosaur_attributes.weight;
+  this.length = dinosaur_attributes.length;
+  this.period = dinosaur_attributes.period;
+}
+
+Dinosaur.prototype.roar = function () {
```

Comment:
Great job using prototypes! Now how would you limit this roar method to only tyrannosaurus?